docs/using_starrocks/Materialized_view.md

```diff
@@ -149,47 +167,38 @@ GROUP BY order_id;
+> - By default, executing a CREATE MATERIALIZED VIEW statement immediately triggers the refresh task, which can consume a certain proportion of the system resources. If you want to defer the refresh task, you can add the REFRESH DEFERRED parameter to your CREATE MATERIALIZED VIEW statement.
 
 - **About refresh mechanisms of asynchronous materialized views**
 
-  Currently, StarRocks supports two ON DEMAND refresh strategies: manual refresh and regular refresh at a fixed time interval.
+  Currently, StarRocks supports two ON DEMAND refresh strategies: MANUAL refresh and ASYNC refresh.
 
-  In StarRocks v2.5, asynchronous materialized views further support a variety of asynchronous refreshing mechanisms:
+  In StarRocks v2.5, asynchronous materialized views further support a variety of asynchronous refreshing mechanisms to control the cost of refresh and increase the success rate:
 
   - If an MV has many large partitions, each refresh can consume a large amount of resources. In v2.5, StarRocks supports splitting refresh tasks. You can specify the maximum number of partitions to be refreshed, and StarRocks performs refresh in batches, with a batch size smaller or equal to the specified maximum number of partitions. This feature ensures large asynchronous materialized views are stably refreshed, enhancing the stability and robustness of data modeling.
   - You can specify the time to live (TTL) for partitions of an asynchronous materialized view, reducing the storage size taken by the materialized view.
   - You can specify the refresh range to refresh only the latest few partitions, reducing the refresh overhead.
+  - You can specify the base tables where data changes will not automatically trigger a refresh of the corresponding materialized view.
+  - You can assign a resource group to the refresh task.
 
   For more information, see the **PROPERTIES** section in [CREATE MATERIALIZED VIEW - Parameters](../sql-reference/sql-statements/data-definition/CREATE%20MATERIALIZED%20VIEW.md#parameters). You can also modify the mechanisms of an existing asynchronous materialized view using [ALTER MATERIALIZED VIEW](../sql-reference/sql-statements/data-definition/ALTER%20MATERIALIZED%20VIEW.md).
 
+  > **CAUTION**
+  >
+  > To prevent exhaustive system resource consumption and potential task failures during full refresh operations, it is recommended to create partitioned materialized views based on partitioned base tables. This ensures that when data updates occur within a base table partition, only the corresponding partition of the materialized view will be refreshed, rather than refreshing the entire materialized view. For more information, please refer to [Data Modeling with Materialized Views - Partitioned Modeling](./data_modeling_with_materialized_views.md#partitioned-modeling).
```

Comment:
1. "exhaustive system resource consumption" is not good. Suggest: To prevent full refresh operations from exhausting system resources and causing task failures, it is xxx   2. will be refreshed --- are refreshed